content/IA-update_connections/Overview.textile

```diff
@@ -0,0 +1,293 @@
+  const ably = new Ably.Realtime.Promise('{{API_KEY}}');
+  await ably.connection.once("connected");
+  console.log('Connected to Ably!');
+
+
+<span lang="ruby">Note that all examples on this page assume you are running them within an EventMachine reactor. Find out more in our "Realtime usage documentation":/realtime/usage.</span>
+
+
+h2(#monitoring-connections). Monitoring connections
+
+p(note). Connections can be monitored from your Ably app dashboard. See the "Ably FAQs":/https://faqs.ably.com/how-do-i-access-my-app-dashboard for more information.
+
+h3(#metachannels). Connection metachannels
+
+Sampled connection events for an app are published to the channel @[meta]clientEvents:connections@. Connection events can be used to compile statistics relating to client population.
+
+The connection event types are:
+
+- connection.opened := The connection was successfully made by the client to Ably.
+- connection.closed := The connection was explicitly closed by the client.
+- connection.refused := The connection was rejected for an expected reason, such as invalid credentials or a malformed request.
+- connection.failed := The connection was rejected for an unexpected reason, such as a network failure.
+
+Events contain a subset of the following metadata in the @data@ field:
+
+- host := The host the connection was made to.<br>__Type: @String@__
+- requestId := The unique ID of the request that Ably can use to correlate a connection event with internal logs, if necessary.<br>__Type: @String@__
+- region := The region the connection was made from.<br>__Type: @String@__
+- headers := The headers sent with the connection.<br>__Type: @JSON Object@__
+- query := The parsed query string of the connection request, excluding authentication parameters. It contains connection information such as the client library version and any custom transport parameters.<br>__Type: @JSON Object@__
+- connectionId := The unique ID of the connection.<br>__Type: @String@__
+- clientId := The ID of the client that attempted the connection.<br>__Type: @String@__
+- channels := A list of channels included in the request. This is only relevant where channels are supported as part of the connection request, such as with "SSE":/sse. <br>__Type: @Array@__
+- duration := The duration the connection was open for.<br>__Type: @Integer@__
+- error := The details of any error encountered whilst making the connection request. It includes an error message, error code and HTTP status code.<br>__Type: @JSON Object@__
+
+An example of a @connection.closed@ event is:
+
+```[json]
+{
+  "host": "realtime.ably.io",
+  "requestId": "fbbcb0ab-fa56-47c4-bbd4-fccc22a271b8",
+  "region": "us-east-1",
+  "headers": {
+    "host": "realtime.ably.io",
+    ...
+  },
+  "query": {
+    "format": "json",
+    "heartbeats": "true",
+    "v": "1.2",
+    "lib": "js-web-1.2.9"
+  },
+  "connectionId": "54321",
+  "clientId": "12345",
+  "duration": 61151
+}
+```
```

Comment:
Shouldn't we also have something on the dev console according to Miro?

Reply:
Done